language/move-prover/tests/sources/stdlib/modules/lbr.move

```diff
@@ -4,10 +4,38 @@ module LBR {
     use 0x0::Transaction;
     use 0x0::Libra;
 
+// here is a change I'm not going to write out
+
     struct T { }
 
     public fun initialize() {
         Transaction::assert(Transaction::sender() == 0xA550C18, 0);
         Libra::register<T>();
     }
+    spec fun initialize {
+        pragma verify = false;
```

Comment:
I guess this is set to false because of the discussed problems. Should we mark such things with a TODO or FIXME so they are easily discoverable?

Reply:
The pragma is unnecessary.  There are some commented out commands that represent problems, so I added "FIXME:" before the comments explaining the problems.